assets/dev/js/editor/utils/jquery-html5-dnd.js

```diff
@@ -318,20 +327,39 @@
 			$( currentElement ).removeClass( settings.currentElementClass );
 
 			self.doDragLeave();
+
+			isDroppingAllowedState = false;
 		};
 
 		var onDrop = function( event ) {
 			setSide( event );
 
-			if ( ! isDroppingAllowed( event ) ) {
+			if ( ! isDroppingAllowedState ) {
 				return;
 			}
 
 			event.preventDefault();
 
-			if ( 'function' === typeof settings.onDropping ) {
-				settings.onDropping.call( this, currentSide, event, self );
-			}
+			const serialized = JSON.stringify( {
+				type: 'section',
+				version: '0.4',
+				content: [
+					elementor.channels.panelElements.request( 'element:selected' )?.model.attributes,
+				],
+			} );
+
+			$e.run( 'document/elements/browser-import', {
```

Comment:
Why not use the simple `document/elements/create` command?

Reply:
I'll make widgets from panel to skip the `browser-import` command next commit.